solidity/core/contracts/AbacusConnectionManager.sol

```diff
@@ -155,7 +168,14 @@ contract AbacusConnectionManager is IAbacusConnectionManager, Ownable {
      */
     function _unenrollInbox(address _inbox) internal {
         uint32 _currentDomain = inboxToDomain[_inbox];
-        domainToInbox[_currentDomain] = address(0);
+        address[] storage inboxes = domainToInboxes[_currentDomain];
+        for (uint256 i = 0; i < inboxes.length; i = i.add(1)) {
```

Comment:
do we really need safemath for this? We iterate in a few places throughout the code and they all don't use SafeMath - I think it's unreasonable to ever have `inboxes.length` be 2^256 or to not run out of gas if it were that long